arch/risc-v/src/bl602/bl602_start.c

```diff
@@ -82,7 +82,7 @@ static struct boot2_partition_table_s g_boot2_partition_table unused_data;
  * Public Data
  ****************************************************************************/
 
-uint32_t g_idle_topstack = 0;
+uintptr_t g_idle_topstack = 0;
```

Comment:
```suggestion
uintptr_t g_idle_topstack;
```

Assigned to 0 by C standard.